src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -7,6 +7,9 @@ Loader {
 
     property string name: ""
 
+    // Badge color properties must be set if badgeItem.visible = true
+    property alias statusSmartIdenticonBadge: statusBadge
```

Comment:
Can just call this `badge` instead. We already know we're in a smart identicon

Reply:
I agree! Done!